spec/features/edit_vacancy_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.feature 'Edit vacancy', type: :feature do
+  context 'with a logged user' do
+    let(:user) { create(:user) }
+
+    scenario 'edit vacancy from my list' do
+      zoo_keeper = create(:vacancy, job_title: 'ZooKeeper', company_id: user.company.id)
+      visit company_vacancies_path(as: user)
+
+      click_link I18n.t('messages.edit_vacancy'), href: edit_vacancy_path(zoo_keeper.id)
```

Comment:
After #86, you should rely on english texts to make the text a little bit better.

```
click_link 'Edit Vacancy'
```

also, you don't need the `href`

Reply:
may have more than one vacancy

Reply:
👍 good point. So, only change to `click_link 'Edit Vacancy'`